aea/cli/registry/utils.py

```diff
@@ -239,8 +264,16 @@ def check_is_author_logged_in(author_name: str) -> None:
     if not author_name == resp["username"]:
         raise click.ClickException(
             "Author username is not equal to current logged in username "
-            "(logged in: {}, author: {}). "
-            "You are allowed to push only items of your authorship.".format(
+            "(logged in: {}, author: {}). Please logout and then login correctly.".format(
                 resp["username"], author_name
             )
         )
+
+
+def is_logged_in():
```

Comment:
I think it should be renamed to `is_auth_token_present` as it is checking this and not whether it is logged in